screens/SettingsScreen.js

```diff
@@ -3,56 +3,67 @@ import { StyleSheet, Text, View } from 'react-native';
 import { CheckBox } from 'react-native-elements';
 import { connect } from 'react-redux';
 import { checkSites } from '../redux/actions/sites';
-import injectionsites from '../components/injectionsites'
+import injectionsites from '../components/injectionsites';
+
+import SignUpModal from '../components/SignUpModal';
+import LoginModal from '../components/LoginModal';
 
 export class SettingsScreen extends React.Component {
   static navigationOptions = {
-    title: 'Settings'
+    title: 'Settings',
   };
 
   onlyUnique(self) {
-    let uniqueParts = []
-    let uniqueSites = []
+    const uniqueParts = [];
+    const uniqueSites = [];
     self.forEach((site, index) => {
-      if ( !uniqueParts.includes(site.part) ) {
-      uniqueParts.push(site.part)
-      uniqueSites.push(site)
+      if (!uniqueParts.includes(site.part)) {
+        uniqueParts.push(site.part);
+        uniqueSites.push(site);
       }
-    })
+    });
     return uniqueSites;
   }
 
   render() {
     return (
-      <View>
-        {
-          this.onlyUnique(this.props.sites).map((cb) => {
-          return (
-            <CheckBox
-              key={cb.part}
-              id={cb.part}
-              title={cb.part}
-              checked={cb.active}
-              onPress={() => this.props.checkSites(cb.part, cb.active)}
-            />
-          )
-        })
-      }
-      </View>
-    )
+      <React.Fragment>
+        <View>
+          {
+            this.onlyUnique(this.props.sites).map(cb => (
+              <CheckBox
+                key={cb.part}
+                id={cb.part}
+                title={cb.part}
+                checked={cb.active}
+                onPress={() => this.props.checkSites(cb.part, cb.active)}
+              />
+            ))
+          }
+        </View>
+        <Text>
+          {'\n'}
+        </Text>
+        <View>
```

Comment:
There are \<View>s inside the modals so think we can delete these. That will help keep it tidy as well.

Reply:
Agreed